packages/3id-did-resolver/src/index.ts

```diff
@@ -107,12 +102,31 @@ const resolve = async (ceramic: Ceramic, didId: string, commit?: string, v03ID?:
 }
 
 export default {
-  getResolver: (ceramic: Ceramic): ResolverRegistry => {
-    return {
-      '3': async (did: string, parsed: ParsedDID): Promise<DIDDocument | null> => {
+  getResolver: (ceramic: Ceramic): ResolverRegistry => ({
+    '3': async (did: string, parsed: ParsedDID, resolver: Resolver, options: DIDResolutionOptions): Promise<DIDResolutionResult> => {
+      const contentType = options.accept || DID_JSON
+      const response: DIDResolutionResult = {
+        didResolutionMetadata: { contentType },
+        didDocument: null,
+        didDocumentMetadata: {}
+      }
+      try {
         const version = getVersion(parsed.query)
-        return isLegacyDid(parsed.id) ? legacyResolve(ceramic, parsed.id, version) : resolve(ceramic, parsed.id, version)
+        const doc = await (isLegacyDid(parsed.id) ? legacyResolve(ceramic, parsed.id, version) : resolve(ceramic, parsed.id, version))
+        if (contentType === DID_LD_JSON) {
```

Comment:
Can you add some comments on what these different cases represent?

Reply:
The contentType?